common/src/event.rs

```diff
@@ -235,12 +240,5 @@ pub enum ProgramChangeKind<BlockNumber> {
     ///
     /// Program is no longer available for interaction, but can be
     /// resumed by paying rent and giving whole data related to it.
-    Paused {
-        /// Code hash the program relates to.
-        code_hash: H256,
-        /// Hash of memory pages of the program.
-        memory_hash: H256,
-        /// Waitlist hash addressed to the program.
-        waitlist_hash: H256,
-    },
+    Paused,
```

Comment:
hash of all data?

Reply:
This is not necessary imho since the event will contain id of the program so all required data could be obtained from storage.

Reply:
yeah, but its better for making sure before calling resume api, furthermore its more comfortable for explorers etc